tigerpath/majors_and_certificates/majors/MOL.json

```diff
@@ -99,11 +99,12 @@
                 {
                   "name": null,
                   "max_counted": 1,
-                  "min_needed": "ALL",
+                  "min_needed": 1,
                   "explanation": "",
                   "completed_by_semester": 4,
                   "course_list": [
-                    "CHM 301"
+                    "CHM 301",
+                    "CHM 303"

```

Comment:
https://molbio.princeton.edu/undergraduate/major/requirements

"Organic Chemistry CHM 301/303 & 302/304/304B, or CHM 337. Courses taken at other institutions can be used toward fulfillment of the chemistry requirements with approval from the Department of Chemistry.  The organic chemistry requirement must be completed before the beginning of the junior year."